Project.toml

```diff
@@ -4,8 +4,11 @@ authors = ["Alexis Montoison (@amontoison), Pierrick Pascal (@ppascal97), Ludovi
 version = "2.0.0"
 
 [deps]
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 NOMAD_jll = "2fc7fd02-576c-576e-a36a-6afce71715da"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Tulip = "6dd1b50a-3aae-11e9-10b5-ef983d2400fa"
 
 [compat]
 julia = "^1.3"
```

Comment:
We should add a compat entry for Tulip.
`Tulip = "^0.7.1"` ?

Reply:
I updated to julia^1.5 equally.